submissions/AnzhelikaKh/popup_task/index.html

```diff
@@ -0,0 +1,122 @@
+                                    <div class="item__view">
+                                        <img class="item__view__img" src="./img/apps.png" alt="">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img class="item__view__img" src="./img/apps.png" alt="">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img class="item__view__img" src="./img/apps.png" alt="">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img class="item__view__img" src="./img/apps.png" alt="">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img class="item__view__img" src="./img/apps.png" alt="">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img class="item__view__img" src="./img/apps.png" alt="">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <input type="checkbox" class="checkbox__more" id="checkbox__more">
+                                <label for="checkbox__more" class="more__label">
+                                    <span class="btn visible">More</span>
+                                </label>
+
+                                <ul class="more__list">
```

Comment:
Houston we have a problem 
![image](https://user-images.githubusercontent.com/16196199/187731136-472423d6-40c8-44c1-8633-c3e2092f6fe6.png)

Reply:
Look like the problem is solved😅